Mark failing tests KNOWNBUG [blocks: #2310, #3627]

tautschnig opened the pull request:
Several tests fail on Windows, but this wasn't noticed due Powershell not aborting on failing commands. Fixes for these tests will follow.

<!---
Thank you for your contribution. Please make sure your pull request fulfils all of the below requirements. If you cannot currently tick all the boxes, but would still like to create a PR, then add the label "work in progress" and assign the PR to yourself.
--->

- [x] Each commit message has a non-empty body, explaining why the change was made.
- n/a Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- n/a The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [x] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- n/a My commit message includes data points confirming performance improvements (if claimed).
- [x] My PR is restricted to a single feature or bugfix.
- n/a White-space or formatting changes outside the feature-related changed lines are in commits of their own.

<!---
See, e.g., https://chris.beams.io/posts/git-commit/ for general guidelines on commit messages.

If you have created commits mixing multiple features and/or unrelated white-space changes, use a sequence involving git reset and git add -p to fix this.
--->

tautschnig committed:
goto-cl does not print source lines

The regression was not running on Windows because of earlier failures.

tautschnig committed:
The first static_assert fails with goto-cl

The regression test is not run on Windows due to earlier failures.

allredj reviewed (commented):
:heavy_check_mark:
Passed Diffblue compatibility checks (cbmc commit: 57eada0d6cf2b4db2839616b80f566852c2db244).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/95900411

tautschnig committed:
Mark tests failing on Windows as KNOWNBUG

All 35 fail with

File: arith_tools.cpp:196 function: from_integer
Condition: false
Reason: Precondition

tautschnig commented:
~~For the record: the fixes for these issues already exist, but I'll refrain from even posting them until #3627 has been resolved as they can't even be tested until then.~~

Fixes in #3636 and #3637.

allredj reviewed (commented):
:heavy_check_mark:
Passed Diffblue compatibility checks (cbmc commit: a3fe0f8dc33dbb47573a63e297d4b27dff2f479f).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/95939500
